docs/index.mdx

```diff
@@ -287,8 +301,55 @@ Note that these identifiers are subject to change and should be updated regularl
           "androidAppId": "ca-app-pub-xxxxxxxx~xxxxxxxx",
           "iosAppId": "ca-app-pub-xxxxxxxx~xxxxxxxx",
           "skAdNetworkItems": [
-            // list of network ids removed to avoid duplication
-            // please refer to the list in the `app.jaon` configuration section above
+            "cstr6suwn9.skadnetwork",
```

Comment:
I read the PR this comment was added and to me it looked like the motivation was to not make the docs page unnecessarily long. Now that we have tabs, I didn't see a reason anymore to not include it in both tabs because the page length is not affected by it anymore. This way users would not be required to switch between the tabs either just the see the IDs.

I wasn't aware we were keeping these ID's updated. Personally, I always check the Google Admob page for these.